yarn-project/sequencer-client/src/block_builder/circuit_block_builder.test.ts

```diff
@@ -253,7 +255,6 @@ describe('sequencer/circuit_block_builder', () => {
       updateVals[0] = 19777494491628650244807463906174285795660759352776418619064841306523677458742n;
       updateVals[1] = 10246291467305176436335175657884940686778521321101740385288169037814567547848n;
 
-      await builder.updateRootTrees();
```

Comment:
Why don't we need to do this anymore?

Reply:
It gets called in the beforeEach, this was a duplicate call

Reply:
it's done in `beforeEach()` already. 
https://github.com/AztecProtocol/aztec3-packages/blob/rk/rollup_cbind_errors/yarn-project/sequencer-client/src/block_builder/circuit_block_builder.test.ts#L69